### DISCUSSION TITLE
Idea: ISC Gathering Lead - an ISC micro-leadership role for third party event

### DISCUSSION THREAD

**Discussion by claredillon** [InnerSourceCommons/InnerSourceMarketing#224] [Ideas]
At the last Marketing WG Call, we discussed the idea of having a new micro-leadership role tasked with bringing together the ISC community at third party events.

**Context:**
- We talked about some of the constraints around organizing InnerSoure Gatherings (cost, time required, ISC commitment up-front  with growing constraints around travel in some organizations => a more risky proposition). 
- Feedback was given about how it was great to have an informal InnerSource Gathering at the last LF OSS US Summit. 
- Benefits included: getting to build relationships with the community, and how doing that just before the event meant that during the event itself each attendee could shape their event attendance for a great InnerSource experience. Tips were given as to which sessions would be great for an InnerSource context, and attendees could arrange more informal meet-ups at breaks and lunch to discuss what they learned. 
- There was zero additional cost to the Foundation or the attendees for this.

**Responsibilities**
So, we discussed the idea that members of the ISC community could nominate themselves as an ISC Gathering Lead. Suggested responsibilities might include:
- Organizing a time and venue for the informal meet-up during, or before the event (early in the event better for max impact for attendees)
- Setting up a channel on Slack event-EventNameYear that people attending can join for realtime updates and chat between the community during the event
- Letting people know the plan on the general channel
- Letting the Marketing WG know so we can send out some stickers and include on the newsletter! :)

**Supports**
We also discussed what the ISC might provide to the ISC Gathering Lead to bring along:
- stickers? - always popular and a low cost - definitely recommended
- Tshirts? Maybe! There is a higher cost, and the down side is that people can (generally) only wear them for one day of the event - so each person would need 2-3 to cover an event. Also - the Tshirts are currently associated with our Summit - and we might like to maintain that association. An alternative to Tshirts we discussed before is...
- Tote bag - same expense as T-shirt - but logistically easier to supply and can be used on multiple days, no sizing required etc.
- We should of course let people know there is a gathering happening through the newsletter.

**Comment by rrrutledge** [comment]
Thank you for writing this up, @claredillon!  One other great idea that @voongc had mentioned is for the lead to be on the lookout for anyone at the conference with InnerSource-type leanings and let them know about the InnerSource Commons and invite them the check out the website and join our _Slack_.

**Comment by claredillon** [comment]
I'll note too - that we need a process for ordering the stickers.

for the ISC Gathering in Dublin in 2022 we used StickerMule - I would recommend them as a vendor for this. The process is:
- Create the order (uploading the image) with instruction
- Approve the proof
- They get shipped. 
Here are the costs for the once we got before as an indication.. we didn't use them all, but have brought them to other events since. 

Minimum costs right now on StickerMule for the holographic stickers is 58 EUR for 50 of the smallest size. An alternative is to have ISC staff get a larger amount and ship them to Gathering Leads as necessary - that would be cheaper in the long run - but would be admin overhead. Perhaps not worth it - considering the low actual cost. 

<img width="614" alt="image" src="https://github.com/InnerSourceCommons/InnerSourceMarketing/assets/32961070/a142f64d-d374-47f8-91dc-eb03b14259e7">

**Reply by claredillon** [reply]
Or at least we should consider when it might make sense to bulk buy - e.g. we are planning to do more than 5 orders a year???
We also need sign off process for costs.

**Reply by rrrutledge** [reply]
Thanks for looking at this, @claredillon.  I'm thinking let's start off with shipping directly to the gathering lead.  If it feels like we're doing it too often then we can switch to larger orders at that time.

**Comment by rrrutledge** [comment]
If we like this, then let's review (at tomorrow's marketing meeting?) and transition it to a GitHub document.

**Comment by rrrutledge** [comment]
I put it in a [pull request (issue template)](https://github.com/InnerSourceCommons/InnerSourceMarketing/pull/230).
